src/main/java/frc/robot/testMode/TestShooterCommand.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot.testMode;
+
+import frc.robot.subsystems.ShooterSubsystem;
+
+public class TestShooterCommand extends TestCommand {
+  private final ShooterSubsystem shooter;
+  private final int targetVelocity;
+  private final boolean turnOffShooter;
+
+  private boolean targetVelocityAcquired = false;
+
+  public TestShooterCommand(ShooterSubsystem shooter, int targetVelocity) {
+    this(shooter, targetVelocity, false);
+  }
+
+  public TestShooterCommand(ShooterSubsystem shooter, int targetVelocity, boolean turnOffShooter) {
+    super();
+
+    this.shooter = shooter;
+    this.targetVelocity = targetVelocity;
+    this.turnOffShooter = turnOffShooter;
+
+    addRequirements(shooter);
+  }
+
+  @Override
+  public void execute() {
+     super.execute();
+
+    shooter.setRawVelocity(targetVelocity);
+  }
+
+  @Override
+  public boolean isFinished() {
+    //finish if we're within 25
+
+    targetVelocityAcquired = Math.abs(shooter.getVelocity() - targetVelocity) < 25;
+
+    return targetVelocityAcquired;
+  }
+
+  @Override
+  public void end(boolean interrupted) {
+    var testModeName = "TestShooterCommand_" + targetVelocity;
+    getTestModeEntry(testModeName).setBoolean(targetVelocityAcquired);
+    getTestModeEntry(testModeName + "_ActionVelocity").setNumber(shooter.getVelocity());
```

Comment:
I like the way you created the NT entry up front in the other tests. If you only create the entry in `end` it won't show up until the test is complete. As it is, it depends on being run with a timeout for a failure to show up.